### ISSUE TITLE
Line breaks with localized formatted strings appears as <br/>

### ISSUE THREAD

**Issue by hichamboushaba** [phrase/phrase-android#14]
We are facing an issue with all formatted strings when using the OTA SDK, they appear as <br/> after updating the SDK to 2.0.0 (before they appeared as \n).

The workaround we are using now is passing the retuned string to Html.fromHtml(), but it's not a pretty solution!

**Comment by theSoenke** [comment]
Hi @hichamboushaba, thanks for reporting it. We are currently looking into this. Could you provide me an example translation for it? That would be very helpful

**Comment by hichamboushaba** [comment]
Hi @theSoenke, any string that has format arguments and line breaks would cause it, for example: 
`"Hello\n%1$s"`

Thanks,

**Comment by theSoenke** [comment]
Thanks for the example. We are currently working on a fix. One way that currently does work is to manually use `String.format` instead of passing format args to `getString`

```
TextView textView = findViewById(R.id.placeholder_key);
String translation = getString(R.string.formatted_string);
textView.setText(String.format(translation, "name"));
```

**Comment by theSoenke** [comment]
We will release a fix for this most likely tomorrow

**Comment by hichamboushaba** [comment]
Thanks a lot for the quick update.

**Comment by hichamboushaba** [comment]
I confirm that the issue is resolved using version 2.0.1.
